### PR TITLE
social page from cartaya

### PR DESCRIPTION
**Pull request by cmcartaya**

### CODE REVIEW

File: cartaya/social-page/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                <p class="socialpg_circle-time">Seconds</p>
+            </div>
+        </div>
+        <form class="socialpg_form" action="#" method="post">
+            <label class="socialpg_text-befor-mail">
+                Provide your email address &amp; we willsend you our newsletter
+            </label>
+            <label class="socialpg_input-mail-container">
+                <img src="./resources/icons/email.svg" alt="mail-icon">
+                <input class="socialpg_input-mail" type="email" name="email" placeholder="Enter your email address">
```
Comment (sarayp) [inline cartaya/social-page/index.html]: need to define an id attribute to the input and bind it to the label for accessibility.... we should always define id and for.

**Review by sarayp — CHANGES_REQUESTED** [review]

File: cartaya/social-page/index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                <p class="socialpg_circle-numbers">60</p>
+                <p class="socialpg_circle-time">Minutes</p>
+            </div>
+            <div class="socialpg_circle">
+                <p class="socialpg_circle-numbers">60</p>
+                <p class="socialpg_circle-time">Seconds</p>
+            </div>
+        </div>
+        <form class="socialpg_form" action="#" method="post">
+            <label class="socialpg_text-befor-mail">
```
Comment (sarayp) [inline cartaya/social-page/index.html]: this label tag is the label and should be the one connected to the one using the _for_ attribute

File: cartaya/social-page/index.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                Provide your email address &amp; we willsend you our newsletter
+            </label>
+            <label class="socialpg_input-mail-container" for="mail">
```
Comment (sarayp) [inline cartaya/social-page/index.html]: This label tag is acting as a container

**Review by sarayp — DISMISSED** [review]

**Review by TahimiLeonBravo — APPROVED** [review]

File: cartaya/social-page/index.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+            </div>
+        </div>
+        <form class="socialpg_form" action="#" method="post">
+            <p class="socialpg_text-befor-mail">
+                Provide your email address &amp; we willsend you our newsletter
+            </p>
+            <label class="socialpg_input-mail-container" for="mail">
+                <img src="./resources/icons/email.svg" alt="mail-icon">
+                <input class="socialpg_input-mail" type="email" name="email" id="mail" placeholder="Enter your email address">
+            </label>
```
Comment (reiniergs) [inline cartaya/social-page/index.html]: there is an issue here with `a11y` when the input gets the focus the screen reader is gonna try to read the text inside the label the input is linked to. there isn't text  with the label here. I assume the text you want to be read is  `Provide your email address &amp; we will send you our newsletter` but there is no clear relationship in between the input and that text.

**Review by reiniergs — CHANGES_REQUESTED** [review]

**Review by reiniergs — APPROVED** [review]
LGTM

**Pull request merged**